src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -43,6 +43,28 @@ public bool WorkerWarmupEnabled
             }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether worker indexing feature is enabled in the hosting config.
+        /// </summary>
+        public bool WorkerIndexingEnabled
+        {
+            get
+            {
+                return GetFeature(RpcWorkerConstants.WorkerIndexingEnabled) == "1";
+            }
+        }
+
+        /// <summary>
+        /// Gets a string delimited by '|' that contains the name of the apps with worker indexing disabled.
+        /// </summary>
+        public string WorkerIndexingDisabledApps
```

Comment:
Where this value about Apps coming from? Will worker/host config mention list of apps disabling indexing?

Reply:
It is fetched from this file. https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Config/FunctionsHostingConfigSource.cs